docs/shortfin/llm/user/e2e_llama8b_mi300x.md

```diff
@@ -22,30 +22,37 @@ python -m venv --prompt shark-ai .venv
 source .venv/bin/activate
 ```
 
-### Install `shark-ai`
+### Install `shortfin`
 
-You can install either the `latest stable` version of `shark-ai`
-or the `nightly` version:
+You can install either the `latest stable` version of shortfin by installing
+`shark-ai` or the `nightly` version directly:
 
 #### Stable
 
 ```bash
-pip install shark-ai
+pip install shark-ai[apps]
 ```
 
 #### Nightly
 
 ```bash
-pip install sharktank -f https://github.com/nod-ai/shark-ai/releases/expanded_assets/dev-wheels
-pip install shortfin -f https://github.com/nod-ai/shark-ai/releases/expanded_assets/dev-wheels
+pip install shortfin[apps] --pre -f https://github.com/nod-ai/shark-ai/releases/expanded_assets/dev-wheels
 ```
 
-#### Install dataclasses-json
+<!-- TODO: Remove when sharktank added to `shark-ai[apps]` -->
+### Install `sharktank`
 
-<!-- TODO: This should be included in release: -->
+Install the `nightly` version of sharktank:
 
 ```bash
-pip install dataclasses-json
+pip install sharktank --pre -f https://github.com/nod-ai/shark-ai/releases/expanded_assets/dev-wheels
+```
```

Comment:
User guides should point to stable releases by default. I'd structure this as

````markdown

## Install stable shark-ai packages

<!-- TODO: Add `sharktank` to `shark-ai` meta package -->

```bash
pip install shark-ai[apps] sharktank
```

### Nightly packages

To install nightly packages:

<!-- TODO: Add `sharktank` to `shark-ai` meta package -->

```bash
pip install shark-ai[apps] sharktank \
    --pre --find-links https://github.com/nod-ai/shark-ai/releases/expanded_assets/dev-wheels
```

See also the
[instructions here](https://github.com/nod-ai/shark-ai/blob/main/docs/nightly_releases.md).

````

(Should check that those install commands all work)

Reply:
Updated commands to match these and verified correct nightly versions are installed